packages/common/src/store/cache/collections/reducer.ts

```diff
@@ -24,7 +24,7 @@ const addEntries = (state: CollectionsCacheState, entries: any[]) => {
   // Add uids to track info in playlist_contents
   // This allows collection tiles to be played when uid would not normally be present
   entries.forEach((entry) => {
-    entry.metadata.playlist_contents.track_ids.forEach(
```

Comment:
wait what.. is this really optional?

Reply:
I undid this change because I couldn't repro the error i got before. but it was definitely weird, i saw a playlist w/ nothing here

Reply:
just to follow up - this was actually broken, but was a manifestation of a deeper issue fixed here: https://github.com/AudiusProject/audius-protocol/pull/9131